diff --git a/GameEngine/Admin/function.php b/GameEngine/Admin/function.php
index 385629d1..bbfed1e6 100755
--- a/GameEngine/Admin/function.php
+++ b/GameEngine/Admin/function.php
@@ -17,9 +17,9 @@
 // # Source code: https://github.com/Shadowss/TravianZ ##
 // # ##
 // ################################################################################
-include_once("../GameEngine/Artifacts.php");
-include_once("../GameEngine/Units.php");
-include_once("../GameEngine/Generator.php");
+include_once("../../GameEngine/Artifacts.php");
+include_once("../../GameEngine/Units.php");
+include_once("../../GameEngine/Generator.php");
 
 class funct
 {
